### PR TITLE
MCKIN-5070: Create apiary documentation for APIs

### PR DESCRIPTION
**Pull request by msaqib52**
This PR documents api-integration APIs, @ziafazal could you please review.

FYI @wajeeha-khalid

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -0,0 +1,4577 @@
+FORMAT: 1A
+HOST: http://api.edx.org
```
Comment (ziafazal) [inline apiary.apib]: Are we going to host these on api.edx.org?

Comment (msaqib52) [inline apiary.apib]: i have removed the host, not required in our case.

**Comment by msaqib52** [comment]
@ziafazal yes it has documentation of all the API endpoints exposed by api-integration

**Comment by wajeeha-khalid** [comment]
@msaqib52 Once merged with edx solutions please update mckinsey fork so that Apiary can sync with apiary.apib and generate doc UI.

**Comment by msaqib52** [comment]
@wajeeha-khalid mckinsey fork is now even with edx-solutions

**Pull request merged**